### PR TITLE
add nSD handling if defined in Plot.xlsx

### PR DESCRIPTION
**Pull request by Felixmil**

### CODE REVIEW

**Comment by PavelBal** [comment]
Since this is not a property the user will find in the plot configuration, I would create a column in the excel template. Additionally, describe in the vignette (and mention the new column in NEWS).

**Comment by Felixmil** [comment]
@PavelBal Do we have enough data in the `TestProject` to plot a `populationTimeProfile` to test aggregation, quantiles and nSD arguments ??

**Comment by PavelBal** [comment]
I think so, there is an example population scenario and plot, if I am not mistaken.

File: tests/testthat/test-create-plots-from-excel.R [hunk 1]
```diff
@@ -855,10 +855,15 @@ test_that("It returns an empty DataCombined when no data is available", {
         "exportConfiguration" = exportConfigurationDfLocal
       ), path = file.path(tempDir, "Plots.xlsx"), )
 
-      dataCombined <- createDataCombinedFromExcel(
-        file = file.path(tempDir, "Plots.xlsx"),
-        stopIfNotFound = FALSE
-      )
+      # Warnings are suppressed because they are expected but not relevant for
```
Comment (PavelBal) [inline tests/testthat/test-create-plots-from-excel.R:858]: Nice, we have tons of warnings in the tests, should be good to suppress them when they are actually expected.

**Comment by Felixmil** [comment]
need https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/1302 to be merge for this to work on Appveyor.

**Comment by Felixmil** [comment]
@Pavel We now have this error:
```
! Could not find an installation of PK-Sim on the machine. Please install the OSPSuite or use 'initPKSim()' to specify the installation path
```

Is it because appveyor is installing latest pksim (v12) while we have ospsuiteR version set to 11.X now ?

**Comment by PavelBal** [comment]
> Is it because appveyor is installing latest pksim (v12) while we have ospsuiteR version set to 11.X now ?

Yes... still discussing with Juri about this, so we have to wait..

File: R/utilities-figures.R [hunk 2]
```diff
@@ -202,8 +205,8 @@ createEsqlabsPlotGridConfiguration <- function() { # nolint: object_length_linte
   plotGridConfiguration <- tlf::PlotGridConfiguration$new()
 
   plotGridConfiguration$tagLevels <- "a"
-  plotGridConfiguration$tagSize <- 10
-  plotGridConfiguration$titleSize <- 11
+  plotGridConfiguration$tagSize <- 11
```
[the remark this answers is not in the document]
Comment (Felixmil) [reply R/utilities-figures.R:208]: I don't find it shocking. See here: https://github.com/esqLABS/esqlabsR/pull/544/files/09027c8899df9efaf919d3f83963a8c5479b72bc#diff-6e0ebe408a5b6c7f7769675226fe011a4b2d00c539afee072e970121a58a9beb

**Review by PavelBal — APPROVED** [review]

**Pull request merged**